salt/master.py

```diff
@@ -2211,7 +2211,11 @@ def _prep_pub(self, minions, jid, clear_load, extra, missing):
 
             if save_load_func:
                 try:
-                    self.mminion.returners[fstr](clear_load['jid'], clear_load, minions=minions)
+                    returner_args = [clear_load['jid'], clear_load]
```

Comment:
IMHO, this whole addition needs some inline documentation to explain the purpose of this code.

Reply:
Yes, this looks good but documentation would be helpful for future maintenance.

Reply:
Looks like this was added back in 2015.8, but was just never documented for save_load in the returner, I am going to revert this change, and just add documentation about it.